src/PrestaShopBundle/Twig/Extension/GridExtension.php

```diff
@@ -142,7 +142,7 @@ public function renderColumnHeader(array $column, array $grid)
             $template = $this->getTemplatePath(
                 $column,
                 $grid,
-                self::BASE_COLUMN_HEADER_TEMPLATE_PATH,
+                $column['options']['header_template_path'] ?? self::BASE_COLUMN_HEADER_TEMPLATE_PATH,
```

Comment:
```suggestion
                $column['options']['header_template_path'],
```
I think we should deprecate self::BASE_COLUMN_HEADER_TEMPLATE_PATH.